tests/unittests/test_metrics.py

```diff
@@ -63,7 +63,8 @@ def test_binary_metrics(device):
     assert summary["FP"] == 1
     assert summary["FN"] == 2
 
-    summary = binary_stats.summarize()
+    summary = binary_stats.summarize(threshold=None)
+    assert summary["threshold"] >= 0.3 and summary["threshold"] < 0.4
```

Comment:
needs also a test for max_samples probably

Reply:
> needs also a test for max_samples probably

I added a test for max_samples=1. Because there are only 3 postive/negative samples in the test case, in this situation, max_samples only work when max_samples==1, and kept 0.1 /0.2for postive/negative samples.

Reply:
Hi @popcornell @mravanelli Should I try to use a for loop in EER function, or we just merge this PR?

Reply:
I think it is fine avoiding for loop...